docs/en/08_troubleshooting.md

```diff
@@ -95,3 +95,127 @@ _-: php: command not found
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 1;
+    private static float $retry_falloff_multiplier_variance = 0;
+
+    // First retry attempt - Retry after 10 minutes
+    // Second retry attempt - Retry after 10 minutes
+    // Third retry attempt - Retry after 10 minutes
+    // Fourth retry attempt - Retry after 10 minutes
+
+    // Exponential retry pattern
+    private static int $max_retry_attempts = 4;
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 2;
+    private static float $retry_falloff_multiplier_variance = 0;
+
+    // First retry attempt - Retry after 10 minutes
+    // Second retry attempt - Retry after 20 minutes
+    // Third retry attempt - Retry after 40 minutes
+    // Fourth retry attempt - Retry after 80 minutes
+
+    // Retry pattern with spread
+    private static int $max_retry_attempts = 4;
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 1;
+    private static float $retry_falloff_multiplier_variance = 0.2;
+
+    // First retry attempt - Retry after 8 to 12 minutes
+    // Second retry attempt - Retry after 6.4 to 14.4 minutes
+    // Third retry attempt - Retry after 5.1 to 27.4 minutes
+    // Fourth retry attempt - Retry after 4 to 38.4 minutes
+}
+```
+
+#### Cluster breaking configuration
+
+This configuration is recommended for dealing with clusters of broken jobs.
+A fixed retry delay typically doesn't help as all jobs will likely be retried in roughly the same time which will repeat the situation that caused the initial cluster to form.
+This scenario is best handled by introducing random delay which spreads the jobs and thus eliminates the cluster.
+It's recommended to refine this configuration in case you have multiple types of jobs that have significantly different priority.
+Higher priority jobs should have lower offset and spread compared to lower priority jobs to minimise waiting times to process high priority jobs.
+
+PHP config
+
+```php
+namespace App\Jobs;
+
+use Symbiote\QueuedJobs\Services\AbstractQueuedJob;
+
+class MyJob extends AbstractQueuedJob
+{
+    private static int $max_retry_attempts = 5;
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 1.2;
+    private static float $retry_falloff_multiplier_variance = 0.2;
+
+    // First retry attempt - Retry after 10 to 19.6 minutes
+    // Second retry attempt - Retry after 10 to 27.4 minutes
+    // Third retry attempt - Retry after 10 to 38.4 minutes
+    // Fourth retry attempt - Retry after 10 to 53.7 minutes
+    // Fifth retry attempt - Retry after 10 to 75.2 minutes
+}
+```
+
+Global configuration is available on the `QueuedJobService` class:
+
+- `job_retry_sentinel` this represents separation of job retry and job processing mechanism to avoid potential edge cases, defaults to 1 minute
```

Comment:
```suggestion
- `job_retry_sentinel` represents separation of job retries and job processing mechanism to avoid potential edge cases, defaults to 1 minute
```